examples/minimal_pub_sub/src/publisher_member_function.rs

```diff
@@ -0,0 +1,59 @@
+use std::{
+    env,
+    sync::{Arc, Mutex},
+    thread,
+    time::Duration,
+};
+
+use anyhow::Result;
+
+struct Publisher {
```

Comment:
Could you name this differently? It'd make it clear that it's not the `Publsher` from `rclrs`. For example,`MinimalPublisher` from the similar `rclcpp` example.